.github/workflows/ci.yml

```diff
@@ -433,6 +433,19 @@ jobs:
               file_put_contents("composer.json", json_encode($j, JSON_PRETTY_PRINT + JSON_UNESCAPED_SLASHES));
             '
 
+            # Ensure that the branch-alias dev-master branch of tractorcow/silverstripe-fluent isn't
+              used instead of 7.x-dev when running kitchen-sink
```

Comment:
```suggestion
            # Ensure that the branch-alias dev-master branch of tractorcow/silverstripe-fluent isn't
            # used instead of 7.x-dev when running kitchen-sink
```

Reply:
or put the comment all on one line - either way's fine

Reply:
Updated